src/systems/filecoin_vm/interpreter/vm_interpreter.go

```diff
@@ -50,71 +51,116 @@ func (vmi *VMInterpreter_I) ApplyTipSetMessages(inTree st.StateTree, msgs TipSet
 			if found {
 				continue
 			}
-			outTree, r = vmi.ApplyMessage(outTree, m, blk.Miner())
+
+			TODO() // TODO: include signature length?
+			onChainMessageLen := len(msg.Serialize_UnsignedMessage(m))
```

Comment:
Now that #669 has merged, please you can discharch this TODO and measure the BLS and SECP message separately.